modular_sand/code/datums/traits/neutral.dm

```diff
@@ -35,17 +35,62 @@
 	desc = "Your system burns with the desire to be bred. Satisfying your lust will make you happy, while ignoring it may cause you to become sad and needy."
 	value = 0
 	mob_trait = TRAIT_ESTROUS_ACTIVE
-	gain_text = span_love("You body burns with the desire to be bred.")
+	gain_text = span_love("You body burns with the desire to engage in breeding.")
 	lose_text = span_notice("You feel more in control of your body and thoughts.")
 
+	// Default heat message for examine text
+	var/heat_type = "influenced by the estrous cycle"
+
+	// Type of interaction to be performed
+	// Intended for use with downstream quirks
+	var/positional_orientation = "engage in breeding"
+
 /datum/quirk/estrous_active/add()
 	// Add examine hook
 	RegisterSignal(quirk_holder, COMSIG_PARENT_EXAMINE, .proc/quirk_examine_estrous_active)
 
+	// Add organ change hooks
+	RegisterSignal(quirk_holder, COMSIG_MOB_ORGAN_ADD, .proc/update_heat_type)
+	RegisterSignal(quirk_holder, COMSIG_MOB_ORGAN_REMOVE, .proc/update_heat_type)
+
 /datum/quirk/estrous_active/remove()
 	// Remove examine hook
 	UnregisterSignal(quirk_holder, COMSIG_PARENT_EXAMINE)
 
+	// Remove organ change hooks
+	UnregisterSignal(quirk_holder, COMSIG_MOB_ORGAN_ADD, .proc/update_heat_type)
+	UnregisterSignal(quirk_holder, COMSIG_MOB_ORGAN_REMOVE, .proc/update_heat_type)
+
+/datum/quirk/estrous_active/post_add()
+	// Update text used by message
+	update_heat_type()
+
+/datum/quirk/estrous_active/proc/update_heat_type()
+	// Check for male reproductive organs
+	var/breed_male = quirk_holder.has_balls()
+
+	// Define possible womb
+	var/obj/item/organ/genital/womb/organ_womb = quirk_holder.getorganslot(ORGAN_SLOT_WOMB)
+
+	// Check for female reproductive organs
+	var/breed_female = (quirk_holder.has_vagina() && istype(organ_womb))
+
+	// Set variable for both reproductive types
+	if(breed_male && breed_female)
```

Comment:
This looks awful but could theoretically pass for now, have you heard about `english_list()`